app/models/protected_area.rb

```diff
@@ -100,6 +105,7 @@ def self.greenlist_coverage_growth(start_year = nil)
       # year = date.to_date.year
       coverage_growth_hash[date] ||= []
       
+      # TODO - ask Ed how exactly the total GL area in the CSV is calculated...
```

Comment:
@stanleypliu Does this mean we would still need to make some changes to the calculations before merging this potentially?

Reply:
So this comment was initially me wondering whether there was an incorrect calculation in the method somewhere because of the difference between the black box data and the coverage chart data. Subsequently having looked into the calculations further, I've found something strange - taking a date at random (I chose 2009), and then assigning `area_sum` to `areas.where("legal_status_updated_at <= ?", date).map(&:name)` I found the following sites: 
```
["Seoraksan", "Odaesan", "Gorgona", "Parco Nazionale Del Gran Paradiso", "Galeras", "Tatama", "Zona marina del Archipiélago de Espíritu Santo", "Chingaza", "Cerbère - Banyuls", "Marais D'Episy", "Pyrénées", "Guadeloupe", "Amarakaeri", "Ajloun Forest Reserve", "Lewa Wildlife Conservancy", "Ol Kinyei", "Cordillera Azul", "Wadi Al-Hitan (Whale Valley)", "Bois Du Loc'H", "Terres Australes Françaises", "Orlu", "Ras Mohammed", "Shouf", "Azraq Oasis", "Vosges Du Nord", "Malpelo", "Isla San Pedro Mártir", "Champ Du Feu", "Hochfeld", "Arakwal", "Montague Island", "Cape Byron", "Iroise", "Ol Pejeta", "Ecrins"]
```
There are no duplicates in that array, but when I then saved that list to `array` and ran `ProtectedArea.where(name: array).count` I got 43, but the length of `array` is 35. I'm suspicious that duplicates are being introduced during summation.

Reply:
After I looked again and mapped the array to WDPA ID instead of name, I found that it matched with the length of the original array (the array that's being reduced to the total gis area) and I don't think it is including duplicates in the summation. It's just something I was afraid of